README.rst

```diff
@@ -109,7 +119,7 @@ https://git-scm.com/docs/git-submodule
 Installing the Images
 =====================
 
-See detailed instructions in doc/howtos/image-install.rst.
+See detailed instructions in `how to install images <doc/howtos/image-install.rst>`__.
```

Comment:
I was experimenting with "anonymous hyperlinks" (where the reference does not create a lingering name that others could reference too).  In this case it doesn't make a difference so I put it back to the regular single underscore.  While this hyperlinking will work on GitHub, this format of hyperlinking would be a problem if we were using Sphinx, but for now it does give us the ability to link between .rst files in GitHub (normally we'd be linking to the generated .html files and not a .rst file).